### PR TITLE
[zh-cn] fix code blocks in 'organizing css'

### LINKED ISSUE #24979: 组织 CSS页面代码错误/表述不清晰
### MDN URL

https://developer.mozilla.org/zh-CN/docs/Learn/CSS/Building_blocks/Organizing

### 与此议题相关的小节或标题？

组织 CSS-将 CSS 格式化成可读的形式

### 哪些信息是错误、无用或不完善的？

代码错误/表述不清晰：

你可以看到很多 CSS 格式化的方式，一些开发者将所有的规则放在一行里面，像是这样：
```
.box {
  background-color: #567895;
}
h2 {
  background-color: black;
  color: white;
}
```

### 你期望看到什么？

你可以看到很多 CSS 格式化的方式，一些开发者将所有的规则放在一行里面，像是这样：

```
.box {background-color: #567895; }
h2 {background-color: black; color: white; }
```

### 你有什么支持的链接、参考或引用？

https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Organizing

### 你还想补充什么内容？

_No response_

### MDN metadata

_No response_

### PR DESCRIPTION
**Pull request by jasonren0403**
### Related issues and pull requests

fixes #24979

### CODE REVIEW

File: files/zh-cn/learn/css/building_blocks/organizing/index.md [hunk 1]
```diff
@@ -126,57 +121,76 @@ CSS 不会管你使用哪种方式来进行格式化，我们自己的看法是
...
 
-...
+/* … */
 ```
 
 然后我们可以加上在整个站点都会用到的所有东西，这可能是像基础页面布局、抬头或者导航栏样式之类的东西。
 
 ```css
-/* || SITEWIDE */
+/* 全站 */
```
Comment (jasonren0403) [inline files/zh-cn/learn/css/building_blocks/organizing/index.md:171]:
```suggestion
/* || 全站 */
```

File: files/zh-cn/learn/css/building_blocks/organizing/index.md [hunk 2]
```diff
@@ -295,24 +309,29 @@ Nicole Sullivan 在描述这种方式和推广上所做的工作导致，即使
...
 <form class="form form--theme-xmas form--simple">
-  <input class="form__input" type="text" />
-  <input class="form__submit form__submit--disabled" type="submit" />
+  <label class="label form__label" for="inputId"></label>
+  <input class="form__input" type="text" id="inputId" />
+
+  <input
+    class="form__submit form__submit--disabled"
+    type="submit"
+    value="Submit" />
```
Comment (yin1999) [inline files/zh-cn/learn/css/building_blocks/organizing/index.md:322]:
```suggestion
    value="提交" />
```

File: files/zh-cn/learn/css/building_blocks/organizing/index.md [hunk 3]
```diff
@@ -295,24 +309,29 @@ Nicole Sullivan 在描述这种方式和推广上所做的工作导致，即使
...
 
 BEM 在大些的 Web 项目中被广泛使用，许多人用这种方式写他们的 CSS。你可能会在没有提及为何 CSS 如此布局的时候，遇到 BEM 语法的例子，甚至是在教程中
 
-阅读[BEM 101](https://css-tricks.com/bem-101/) 中关于 CSS Tricks 的段落以了解更多和这个系统相关的信息。
+阅读 [BEM 101](https://css-tricks.com/bem-101/) 中关于 CSS Tricks 的段落以了解更多和这个系统相关的信息。
 
 #### 其他常见体系
 
-应用中，有很多这样的体系。其他流行的方式包括 Jonathan Snook 创造的[Scalable and Modular Architecture for CSS (SMACSS)](http://smacss.com/)、Harry Roberts 的[ITCSS](https://itcss.io/)以及原本由 Yahoo! 创造的[Atomic CSS (ACSS)](https://acss.io/)。如果你遇到了使用这几种方式之一的项目，那么好处就是你可以搜索到许多文章和指导，帮你理解如何以同种方式写代码。
+应用中，有很多这样的体系。其他流行的方式包括 Jonathan Snook 创造的 [CSS 的可扩展模块化架构（SMACSS）](https://smacss.com/)、Harry Roberts 的[ITCSS](https://itcss.io/)以及原本由 Yahoo! 创造的[原子 CSS（ACSS）](https://acss.io/)。如果你遇到了使用这几种方式之一的项目，那么好处就是你可以搜索到许多文章和指导，帮你理解如何以同种方式写代码。
```
Comment (yin1999) [inline files/zh-cn/learn/css/building_blocks/organizing/index.md:334]:
```suggestion
应用中，有很多这样的体系。其他流行的方式包括 Jonathan Snook 创造的 [CSS 的可扩展模块化架构（SMACSS）](https://smacss.com/)、Harry Roberts 的 [ITCSS](https://itcss.io/) 以及原本由 Yahoo! 创造的[原子 CSS（ACSS）](https://acss.io/)。如果你遇到了使用这几种方式之一的项目，那么好处就是你可以搜索到许多文章和指导，帮你理解如何以同种方式写代码。
```

File: files/zh-cn/learn/css/building_blocks/organizing/index.md [hunk 4]
```diff
@@ -357,14 +376,14 @@ $base-color: #c6538c;
...
+如果你对加入例如许多额外的注释和空格，增大你的样式表大小有所关心的话，那么后处理会通过在生产版本中略去任何不必要的东西的方式，优化 CSS。后处理解决方案中，通过这种方式实现的一个例子是 [cssnano](https://cssnano.github.io/cssnano/)。
 
 ## 结语
 
 这是我们学习 CSS 的指导的最后一部分，正如你所见，你可以通过许多方式从这里继续对 CSS 的探索。
 
 欲了解更多关于 CSS 布局的事情，查看[学习 CSS 布局](/zh-CN/docs/Learn/CSS/CSS_layout)这节课。
 
-你应该也有了探索[MDN CSS](/zh-CN/docs/Web/CSS)学习材料的剩余部分的技能。你可以查阅属性和值，探索我们的[CSS Cookbook](/zh-CN/docs/Web/CSS/Layout_cookbook)来了解可用的排布，在诸如[Guide to CSS Grid Layout](/zh-CN/docs/Web/CSS/CSS_grid_layout)的一些更加专门的指导里阅读更多。.
+你应该也有了探索 [MDN CSS](/zh-CN/docs/Web/CSS) 学习材料的剩余部分的技能。你可以查阅属性和值，探索我们的 [CSS 指导手册](/zh-CN/docs/Web/CSS/Layout_cookbook)来了解可用的布局模式，在诸如 [CSS 栅格布局](/zh-CN/docs/Web/CSS/CSS_grid_layout)的一些更加专门的指导里阅读更多。
```
Comment (yin1999) [inline files/zh-cn/learn/css/building_blocks/organizing/index.md:387]:
```suggestion
你应该也有了探索 [MDN CSS](/zh-CN/docs/Web/CSS) 学习材料的剩余部分的技能。你可以查阅属性和值，探索我们的 [CSS 指导手册](/zh-CN/docs/Web/CSS/Layout_cookbook)来了解可用的布局模式，或继续阅读如 [CSS 网格布局指南](/zh-CN/docs/Web/CSS/CSS_grid_layout)等更加专门的指南。
```

**Review by yin1999 — COMMENTED** [review]
some suggestion

**Pull request merged**